charts/k8ssandra/values.yaml

```diff
@@ -8,19 +8,21 @@ cassandra:
   #    - 3.11.10
   #    - 3.11.11
   #    - 4.0.0
+  #    - 4.0.1
   version: "4.0.0"
```

Comment:
Should we bump the default to 4.0.1 as well?

I'd say this would be an otherwise contentious discussion, but it's just a patch release, so perhaps makes sense?  What does everyone @k8ssandra/maintainers think?

Reply:
If it's a bugfix release, why wouldn't we bump?

Reply:
Agreed, we should bump. Good practice being that peeps set it explicitly in their deployments, which allows them to control when an upgrade should happen.

Reply:
I agree, we should 👍

Reply:
so, they only catch with updating it here is potentially the integration tests. Same issue when we added 3.11.11. The upgrade tests fail because "4.0.1" isn't a valid Cassandra version in older K8ssandra chart versions.

I can update the default here and maybe leave the integration test matrix at 4.0.0 to see how the tests go, but we will have to address the tests at some point.